podman/module-artifact-gen/podman-artifact-gen

```diff
@@ -0,0 +1,177 @@
+#!/bin/sh
+
+set -e
+
+show_help() {
+  cat << EOF
+
+Simple tool to generate Mender Artifact suitable for podman Update Module
+
+Usage: $0 [options] IMAGE [IMAGES...] [-- [options-for-mender-artifact] ]
+
+    Options: [ -n|artifact-name -t|--device-type --software-name --software-version --software-filesystem --run-args -o|--output_path -h|--help ]
+
+        --artifact-name       - Artifact name
+        --device-type         - Target device type identification (can be given more than once)
+        --software-name       - Name of the key to store the software version: rootfs-image.NAME.version,
+                                instead of rootfs-image.podman.version
+        --software-version    - Value for the software version, defaults to the name of the artifact
+        --software-filesystem - If specified, is used instead of rootfs-image
+        --run-args            - Command line args to add when running the container(s)
+        --output-path         - Path to output file. Default: podman-artifact.mender
+        --help                - Show help and exit
+        IMAGE [IMAGES...]     - Podman container images to add to the Artifact
+
+Anything after a '--' gets passed directly to the mender-artifact tool.
+
+EOF
+}
+
+show_help_and_exit_error() {
+  show_help
+  exit 1
+}
+
+check_dependency() {
+  if ! which "$1" > /dev/null; then
```

Comment:
I'd suggest using `command`, `which` may itself not be available.

Reply:
Do you mean something like:
`if ! command -v "$1" > /dev/null; then`

Then, the other modules might need to be modified too ? docker, k8s , etc. are all using which.

Reply:
You're right. But there is `command` used that way in this PR too so maybe it's time to set the right direction!